akka-docs/rst/general/stream/stages-overview.rst

```diff
@@ -149,6 +149,20 @@ broadcast              all of the outputs stops backpressuring and there is an i
 balance                any of the outputs stops backpressuring; emits the element to the first available output                                    all of the outputs backpressure                                                                                                 upstream completes
 =====================  =========================================================================================================================   ==============================================================================================================================  =====================================================================================
 
+Watching status stages
+^^^^^^^^^^^^^^^^^^^^^^
+
+There is only one stage available now that watching for completion/cancellation.
+It completes Future successfully when received complete message from upstream or cancel
```

Comment:
"Materializes to a Future that will be completed with Unit or failed depending whether the upstream of the stage has been completed or failed. The stage otherwise passes through elements unchanged."